Date: 2005-10-19
Initial Package Version: 2.9.7
Upstream Status: Not yet submitted
Origin: Miklos Vajna <vmiklos@frugalware.org>
Description: Fixes ownership problems (fst:users instead of root:root) when:
 - building in a chroot
 - using fakeroot
 - using --sudosync to install packages
 makepkg |    4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)
diff -Naur pacman-2.9.7.orig/doc/pacman.8.in pacman-2.9.7/doc/pacman.8.in
--- pacman-2.9.7.orig/scripts/makepkg.orig	2005-09-18 00:43:58.000000000 +0200
+++ pacman-2.9.7/scripts/makepkg	2005-09-18 02:11:51.000000000 +0200
@@ -142,7 +142,7 @@
 			fi
 			sudo pacman -D $deplist
 			if [ "$INFAKEROOT" = "1" ]; then
-				FAKEROOTKEY=$FAKEROOTKEY2
+				export FAKEROOTKEY=$FAKEROOTKEY2
 				unset FAKEROOTKEY2
 			fi
 			if [ "$?" = "127" ]; then
@@ -1203,7 +1202,7 @@
 	fi
 	sudo pacman -R $makedeplist $deplist
 	if [ "$INFAKEROOT" = "1" ]; then
-		FAKEROOTKEY=$FAKEROOTKEY2
+		export FAKEROOTKEY=$FAKEROOTKEY2
 		unset FAKEROOTKEY2
 	fi
 fi
